artman/config/proto/config.proto

```diff
@@ -266,6 +252,7 @@ message Artifact {
   // [dependencies.yaml](https://github.com/googleapis/googleapis/blob/master/gapic/packaging/dependencies.yaml)
   PackageVersion package_version = 14;
 
+  // DEPRECATED.
```

Comment:
Maybe use `UNSUPPORTED` (here and below)

Reply:
Done.